src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -64,50 +60,53 @@
 import edu.wpi.first.wpilibj2.command.Commands;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants.CtreCanID;
+import frc.robot.Constants.DIOConstants;
 import frc.robot.Constants.RevCanID;
 import frc.robot.Robot;
 import utilities.DebugEntry;
 import utilities.TOFSensorSimple;
 
 public class IntakeSubsystem extends SubsystemBase {
   /** Creates a new IntakeSubsystem. */
-  private SparkMax intakeMotor;
+  private TalonFX intakeMotor;
 
   private TalonFXSimState simPivotMotor;
-  private CANcoderSimState simCANcoder;
 
   private TalonFX pivotMotor;
-  private SparkMax conveyorMotor;
+  private TalonFX conveyorMotor;
   private Angle pivotSetpoint = kPivotRetractAngle;
 
+  private DutyCycleEncoder throughBoreEncoder;
+
+  // Sensor closest to birdhouse
   private TOFSensorSimple sensor;
   private SimDeviceSim simSensor;
   private SimBoolean simbeam;
+
   private Mechanism2d mech = new Mechanism2d(2, 2);
   private ComplexWidget widget;
   private MechanismLigament2d pivotArmMech;
 
-  enum intakeState {
+  private enum IntakeState {
     Idle,
     atSensor,
     coralLoaded
   }
 
-  intakeState state = intakeState.Idle;
-  Timer timer = new Timer();
+  private IntakeState state = IntakeState.Idle;
+  private Timer timer = new Timer();
 
   private SingleJointedArmSim pivotSim;
 
   private DebugEntry<Double> pivotOutput;
   private DebugEntry<String> currentCommand;
 
-  private CANcoder CANcoder;
-
   public IntakeSubsystem() {
-    intakeMotor = new SparkMax(RevCanID.kIntakeMotor, MotorType.kBrushless);
+    intakeMotor = new TalonFX(CtreCanID.kIntakeMotor);
     pivotMotor = new TalonFX(CtreCanID.kPivotMotor);
-    conveyorMotor = new SparkMax(RevCanID.kConveyorMotor, MotorType.kBrushless);
+    conveyorMotor = new TalonFX(CtreCanID.kConveyorMotor);
     sensor = new TOFSensorSimple(RevCanID.kConveyorSensor, Inches.of(1));
```

Comment:
There should be a new version of this class in a different commit of saddlebags for supporting LaserCAN. Ask @ModBoyex (Jackson) for details.